gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceConnector.java

```diff
@@ -64,8 +64,10 @@ public SalesforceConnector(State state) {
   @Override
   public HttpEntity getAuthentication() throws RestApiConnectionException {
     log.debug("Authenticating salesforce");
-    String clientId = this.state.getProp(ConfigurationKeys.SOURCE_CONN_CLIENT_ID);
-    String clientSecret = this.state.getProp(ConfigurationKeys.SOURCE_CONN_CLIENT_SECRET);
+    String clientId = PasswordManager.getInstance(this.state)
```

Comment:
Can we have this decryption logic configurable? So that not all the case we need this decyption. Today BDE team is also relying on this code base.

Reply:
Considering to reuse the same returned variable from PasswordManager.getInstance(this.state)